### ISSUE TITLE
Goerli prod tests

### ISSUE THREAD

**Issue by eternauta1337** [Synthetixio/synthetix-pm#126]
Interesting problem:
* Prod tests are currently tuned to mainnet.
* Running prod tests on Goerli fails, because the system is currently suspended on Goerli.
* So what do we do? Do per network specific prod tests? Handle a lot of conditionals per-network in prod tests?
* My solution was to spin up a fork and use the interactive ui to unsuspend the system, then run the prod tests.
* Most of the tests failed anyway because no oracles are set in Goerli. However, by commenting some parts of the tests, I was able to verify that ERC20 transfers work. 

So, in conclusion, all the Goerli deployment is, is an ERC20 with balances, and the SecondaryDeposit contract. And that works.
